integration-tests/contract-transfer-ticket-between-implicit-and-originated-accounts.spec.ts

```diff
@@ -0,0 +1,102 @@
+
+CONFIGS().forEach(({ lib, protocol, rpc, setup, createAddress }) => {
+  const Tezos1 = lib;
+  const client = new RpcClient(rpc);
+  const mumbaiAndAlpha = protocol === Protocols.PtMumbaii || protocol === Protocols.ProtoALpha ? test : test.skip;
+  let tezos1Pkh: string;
+  let Tezos2: TezosToolkit;
+  let ticketSendContract: DefaultContractType;
+  let ticketBagContract: DefaultContractType;
+  let ticketBlackholeContract: DefaultContractType;
+  let ticketToken: TicketTokenParams
+
+  describe(`Test contract origination of a simple contract through contract api using: ${rpc}`, () => {
+
+    beforeAll(async (done) => {
+      await setup(true);
+      try {
+        Tezos2 = await createAddress();
+        tezos1Pkh = await Tezos1.signer.publicKeyHash();
+
+        const ticketSendOrigination = await Tezos1.contract.originate({ code: ticketsSendTz, storage: null});
+        await ticketSendOrigination.confirmation();
+        ticketSendContract = await ticketSendOrigination.contract();
+
+        const ticketBagOrigination = await Tezos1.contract.originate({ code: ticketsBagTz, storage: []});
+        await ticketBagOrigination.confirmation();
+        ticketBagContract = await ticketBagOrigination.contract();
+
+        const ticketBlackholeOrigination = await Tezos1.contract.originate({ code: ticketsBlackholeTz, storage: null});
+        await ticketBlackholeOrigination.confirmation();
+        ticketBlackholeContract = await ticketBlackholeOrigination.contract();
+
+        ticketToken = { ticketer: ticketSendContract.address, content_type: { prim: 'string' }, content: { string: 'Ticket' } };
+      } catch(error){
+        console.error(error);
+      }
+      done();
+    });
+
+    describe('transfer tickets between implicit and originated accounts', () => {
+
+        mumbaiAndAlpha('will send 3 tickets from an originated to an implicit account', async (done) => {
+            const ticketSendToImplicitOp = await ticketSendContract.methods.default(tezos1Pkh, '3').send();
+            await ticketSendToImplicitOp.confirmation();
+            expect(ticketSendToImplicitOp.status).toEqual('applied');
+
+            let tezos1TicketBalance = await client.getTicketBalance(tezos1Pkh, ticketToken );
+            expect(tezos1TicketBalance).toBe('3');
+            done();
+        });
+
+        mumbaiAndAlpha('will transfer 1 tickets from an implicit to another implicit account', async(done) => {
+          const implicitToImplicitOp = await Tezos1.contract.transferTicket({
+              ticketContents: { string: "Ticket" },
+              ticketTy: { prim: "string" },
+              ticketTicketer: ticketSendContract.address,
+              ticketAmount: 1,
+              destination: await Tezos2.signer.publicKeyHash(),
+              entrypoint: 'default',
+          });
+          await implicitToImplicitOp.confirmation();
+          expect(implicitToImplicitOp.status).toEqual('applied');
+
```

Comment:
can we add an assertion to check if Tezos2 was 0 before it was transferred, and maybe another assert to check whether Tezos1's balance is reduced?